dockerfiles/stages/3-toolchain

```diff
@@ -24,55 +24,32 @@ RUN apt-get update --yes \
     apt-transport-https \
     apt-utils \
     awscli \
-    ca-certificates \
```

Comment:
It seems this commit contains changes from https://github.com/MinaProtocol/mina/pull/17311 as well?

Reply:
Yes, my bad at splitting PRs